### ISSUE TITLE
Performance problems due to postMessage?

### ISSUE THREAD

**Issue by schwichti** [kieler/elkjs#75]
Hi,
when I use elkjs my react app slows down significatly. I opened the profiler and noted that postMessage seems to be quite expensive.
![image](https://user-images.githubusercontent.com/33393719/79315264-fd9fec80-7f02-11ea-8c43-e010d921f598.png)

Do you have any hints what I can do about that?

**Comment by uruuru** [comment]
Is this for a small or a large graph? The layout of larger graphs and certain layout algotihms can take quite some time.

**Comment by schwichti** [comment]
I had to mention that I have started the profiler **AFTER** the layout was calculated and applied.

When I had this issue I was using elkjs inside a React functional component. In the meanwhile I refactored them into React class components (because of another problem). A nice side-effect of the refactoring is that I do not have the performance issue anymore. I checked the profiler again, and the postMessage calls also seem to be gone now.
